src/js/actions/shapes.js

```diff
@@ -160,27 +161,27 @@ define(function (require, exports) {
      * @param {Document} document
      * @param {Immutable.List.<Layer>} layers list of layers being updating
      * @param {Stroke} stroke Stroke properties to apply
-     * @param {boolean=} enabled
+     * @param {boolean=} options.enabled Default true
      * @return {Promise}
      */
-    var setStroke = function (document, layers, stroke, enabled) {
+    var setStroke = function (document, layers, stroke, options) {
         // if enabled is not provided, assume it is true
         // derive the type of event to be dispatched based on this parameter's existence
-        var eventName,
-            enabledChanging;
-        if (enabled === undefined || enabled === null) {
-            enabled = true;
+        var eventName;
+
+        if (options.enabled === undefined || options.enabled === null) {
+            options.enabled = true;
             eventName = events.document.history.optimistic.STROKE_CHANGED;
         } else {
             eventName = events.document.STROKE_ENABLED_CHANGED;
-            enabledChanging = true;
```

Comment:
Since a this if/else is applying the defaulting logic, doesn't it still need to set the enabledChanging bit?